src/pages/tickets.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<!--head section-->
+<head>
+  <title>Shapeland - Tickets</title>
+  <meta charset="utf-8">
+  <meta name="description" content="Practice web page.">
+  <meta name="author" content="shapeland">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="refresh" content="800">
+  <meta name="keywords" content="uol theme park">
+  <link rel="stylesheet" type="text/css" href="../css/main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> <!--social favigon-->
+  <link rel="stylesheet" type="text/css" href="../css/tickets.css">
+</head>
+
+<!--Ticket page body-->
+<body>
+    <!--header section-->
+	
+    <!--navbar--> 
+    <header class="sticky" style="z-index: 9999;"> 
+        <nav id="navbar">
+            <nav class="nav-container">
+                <div class="logo-container"><img src="../public/images/shape_logo_circ.png" class="logo"></div>
+                <div class="menu-container"><b>Attractions &amp; Facilities</b></div>
+                <div class="menu-container"><b>Ticket info</b></div>
+            </nav>
+        </nav>
+    </header>
+	
+	<!--hero img-->
+	<header class="hero-image">
+		<div class="hero-text">
+			Tickets
+		</div>
+	</header>
+
+	<!--announcements-container below header image-->
+	<section class="announcements-container">
+		<div style="flex-grow: 1">One ticket gives access to all attractions!</div>
+		<div style="flex-grow: 4"></div>
+		<div style="flex-grow: 0.1">Purchase in App and save 10%!</div>
+	</section>
+
+    <!--main-->
+    <main id="main-container">
+        <!--ticket information-->
+        <section class="ticket-container">
+            <div class="ticket-label">
+                <div><h1>Option One</h1></div>
+                <div><span><b>Information:</b></span></div>
+                <div><span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ac facilisis metus. 
+                    Donec faucibus vel lorem vitae cursus. 
+                    Integer vel feugiat sapien, sed tempor augue.
+                    ras ut sapien eu arcu convallis accumsan.
+                    In eget ante et est sollicitudin venenatis. Duis nec euismod odio, non ultrices nibh..</span></div>
+                <div class="price"><h1>$10</h1></div>
```

Comment:
You could try to move the `class`  to the `h1`  tag, see if it behaves accordingly and if the display is not the same you could try setting `display: block` on the styling for `price`.